atmel-rf-driver/NanostackRfPhyAtmel.h

```diff
@@ -52,14 +55,33 @@
 #if !defined(ATMEL_I2C_SCL)
 #define ATMEL_I2C_SCL    D15
 #endif
+#if !defined(ATMEL_SPI_TEST1)
+#define ATMEL_SPI_TEST1    D6
+#endif
+#if !defined(ATMEL_SPI_TEST2)
+#define ATMEL_SPI_TEST2    D3
+#endif
+#if !defined(ATMEL_SPI_TEST3)
+#define ATMEL_SPI_TEST3    D4
+#endif
+#if !defined(ATMEL_SPI_TEST4)
+#define ATMEL_SPI_TEST4    D2
+#endif
+#if !defined(ATMEL_SPI_TEST5)
+#define ATMEL_SPI_TEST5    D8
+#endif
 
 class RFBits;
 
 class NanostackRfPhyAtmel : public NanostackRfPhy {
 public:
     NanostackRfPhyAtmel(PinName spi_mosi, PinName spi_miso,
                         PinName spi_sclk, PinName spi_cs,  PinName spi_rst, PinName spi_slp, PinName spi_irq,
-                        PinName i2c_sda, PinName i2c_scl);
+                        PinName i2c_sda, PinName i2c_scl
```

Comment:
Those I2C pins are very platform independent feature and not direct about the radio specific. Let's